server/database/fakeData.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO user(username , email , password) VALUES ('Alaa' , 'alaa@gmail.com' , '12345678');
```

Comment:
You should store the hash of the password instead of storing plain text passwords in the database .